src/common/com/intellij/plugins/haxe/util/HaxePsiUtils.java

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
For this file, I'm not sure why you wanted to create a new set of utility functions, when they already exist.  Most of them already exist in UsefulPsiTreeUtil and PsiTreeUtil.